src/status_im/ui/screens/discover/events.cljs

```diff
@@ -27,26 +29,30 @@
                                       [message-id discover]))
                                (into {})))))
 
+;; todo(goranjovic): at the moment we do nothing when a status without hashtags is posted
+;; but we probably should post a special "delete" status that removes any previous
+;; hashtag statuses in that scenario
 (register-handler :broadcast-status
   (u/side-effect!
     (fn [{:keys [current-public-key web3]
           :accounts/keys [accounts current-account-id]
           :contacts/keys [contacts]}
-         [_ status hashtags]]
-      (let [{:keys [name photo-path]} (get accounts current-account-id)
-            message-id (random/id)
-            message    {:message-id message-id
-                        :from       current-public-key
-                        :payload    {:message-id message-id
-                                     :status     status
-                                     :hashtags   (vec hashtags)
-                                     :profile    {:name          name
-                                                  :profile-image photo-path}}}]
-        (doseq [id (u/identities contacts)]
-          (protocol/send-status!
-            {:web3    web3
-             :message (assoc message :to id)}))
-        (dispatch [:status-received message])))))
+         [_ status]]
+      (if-let [hashtags (seq (handlers/get-hashtags status))]
+        (let [{:keys [name photo-path]} (get accounts current-account-id)
+              message-id (random/id)
+              message    {:message-id message-id
+                          :from       current-public-key
+                          :payload    {:message-id message-id
+                                       :status     status
+                                       :hashtags   (vec hashtags)
+                                       :profile    {:name          name
+                                                    :profile-image photo-path}}}]
+          (doseq [id (u/identities contacts)]
```

Comment:
Do we want to be able to perform those calls at once? Even if current implementation synchronously loop over each contact, some other implementation might offer optimizations.
That would require to add something like `send-statuses!` to protocol.

Reply:
That's a good question, but it's not how it works in the current implementation and the scope of this PR is to fix something that did work and then got broken. Keep in mind, we're not talking about sending multiple statuses to the same contact, but sending the same status to multiple contacts - I wonder if there is anything in whisper to address this.

Reply:
Right, let's ignore this for now.

Reply:
But it is a good question, and to be honest we may only get the answers by experimenting. E.g. what is better - individual messages or bulk messages? We don't want to flood the network, but then again whisper penalizes us for too large payloads, so where's the sweet spot...

Reply:
We might have more than `whisper` in the future so it's always great to anticipate. And a bulk method can always be used for a single contact :)